test/__tests__/wallet.test.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const wallet = require('../../services/wallet');
+const mocks = require('../__mocks__/wallets');
+const request = require('../../services/request');
+
+describe('createWallet', () => {
+  it('Returns an object with 3 properties', () => {
+    return expect(Object.values(wallet.createWallet()).length).toEqual(3)
+  });
+
+  it('Properties should be string type', () => {
+    const properties = Object.values(wallet.createWallet())
+    return expect(properties.every(prop => typeof prop === 'string')).toBe(true)
+  });
+})
+
+describe('getWalletBalance', () => {
+  it('Returns a number', async () => {
+    const response = await wallet.getWalletBalance(mocks.wallet1.address);
+    return expect(typeof response).toBe('number')
+  });
+
+  /** This test could start failing if the testingWallet receives a transaction. */
+  it('Returns the total number of satoshis in the wallet', async () => {
+    const response = await wallet.getWalletBalance(mocks.testingWallet.address);
+    return expect(response).toBe(21000)
+  })
+
+  it('Logs an error if argument is not a correct public addres', async () => {
+    const spy = jest.spyOn(global.console, 'error')
+
+    const response = await wallet.getWalletBalance('not an addres!');
+    expect(spy).toHaveBeenCalled();
```

Comment:
I think you should restore `global.console` for it to be the original one again.